### DISCUSSION TITLE
Alterar diretório dos arquivos Base

### DISCUSSION THREAD

**Discussion by brunoflma** [rictom/rede-cnpj#27] [General]
Olá como vai? Devido ao tamanho absurdo das bases de dados da Receita e a constante necessidade de atualização delas onde teriam que ser sobrescritas estou querendo utilizar os arquivos em outro diretório que não esteja dentro da própria pasta rede, mas o tanto o arquivo rede.ini quanto o rede_sqllite_cnpj.py continuam entendendo que o caminho é sempre direcionado dentro da pasta rede.
Como eu poderia alterar o diretório da base de dados para uma pasta ou até mesmo uma unidade diferente de onde se encontra o programa?

**Comment by rictom** [comment]
Olá @brunoflma 
Sugiro vc colocar o caminho inteiro, a partir da letra do drive, no arquivo de configuração rede.ini:
base_rede =D:/pasta_com_dados/rede.db
É bom usar apenas / ao invés de \\.
Depois de salvar o rede.ini, tem que reiniciar o rede.py

**Reply by brunoflma** [reply]
Olá @rictom agradeço a rapidez na resposta. De fato no Windows bastou colocar o caminho absoluto e mesmo assim ele reconheceu o local das bases, mas no Linux que é atualmente minha base de produção eu não consigo de forma alguma, consegue me ajudar nesse cenário?

**Reply by rictom** [reply]
Oi @brunoflma 
Fica difícil saber sem mais informações. Vc pode colar o que aparece de erro no console?
